### DISCUSSION TITLE
Problems integrating ner component to an existing component

### DISCUSSION THREAD

**Discussion by Alvaro8gb** [explosion/spaCy#11033] [Help: Coding & Implementations] [answered]
Is there any way to integrate the spacy es_core_news_lg pretrained model with a component of our own, since when doing it with add_pipe the tok2vec module is different and predicts wrongly.

**Comment by polm** [comment, marked as answer]
You should be able to do this using `replace_listeners` to bundle your NER component with its tok2vec. Take a look at the [Double NER sample project](https://github.com/explosion/projects/tree/v3/tutorials/ner_double), and let us know if that doesn't work.
